tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/report.py

```diff
@@ -0,0 +1,40 @@
+import time
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import Base
+from gaiatest.form_controls.header import GaiaHeader
+from gaiatest.tests.functional.messages.test_sms_report import TestSmsReport
```

Comment:
Side note about the responsibilities: A page class shouldn't be aware of which tests are using them. The usage diagram is the following:
```
Test -----> Page class -----> Marionette
```
This mean that, marionette is used by the page class, but doesn't import any page class. Same thing between the page class and the test. This also means that the test doesn't use marionette directly, it's always done by the page class.